packages/atomic/src/components/commerce/atomic-commerce-query-summary/e2e/atomic-commerce-query-summary.e2e.ts

```diff
@@ -0,0 +1,75 @@
+import {test, expect} from './fixture';
+
+test.describe('when search has not been executed', () => {
+  test.beforeEach(async ({page}) => {
+    await page.goto(
+      'http://localhost:4400/iframe.html?id=atomic-commerce-query-summary--default&viewMode=story'
```

Comment:
You should be able to use `await page.load()` when up to date with master now.

Or `await page.load({}, 'with-search-box')` etc.

Reply:
- [ ] ☝️ this